src/functionaltests/resources/features/authentication.feature

```diff
@@ -0,0 +1,29 @@
+#Author: valentin.tyhonov@ericsson.com
+#Keywords Summary :
+#Feature: List of scenarios.
+#Scenario: Business rule through list of steps with arguments.
+#Given: Some precondition step
+#When: Some key actions
+#Then: To observe outcomes or validation
+#And,But: To enumerate more Given,When,Then steps
+#Scenario Outline: List of steps for data-driven as an Examples and <placeholder>
+#Examples: Container for s table
+#Background: List of steps run before each of the scenarios
+#""" (Doc Strings)
+#| (Data Tables)
+#@ (Tags/Labels):To group Scenarios
+#<> (placeholder)
+#""
+## (Comments)
+#Sample Feature Definition Template
+Feature: Test Authentication
+
+  Scenario: Call an REST API without credentials
+    Given LDAP is activated
+    When make a POST request to the subscription REST API "/subscriptions" without credentials
+    Then get response code of 401 and subscription with name "Subscription_Test" is not created
+
+	Scenario: Call an REST API with credentials
+		Given LDAP is activated
+		When make a POST request to the subscription REST API "/subscriptions" with username "gauss" and password "password"
```

Comment:
Maybe you should have SubscriptionName "Subscription_Test" mentioned as input in when step also. Or do you use the template file with the existing defined SubscriptionName.

Reply:
Should it not be @tag1 @tag2 before the scenarios also? Thats how Vali did it.

Reply:
I use template with existed subscriptionName. Actually I can try to implement name changing and pass subscriptionName as an input parameter if it is needed.

What about tags, not sure for what they are needed.

Reply:
@tag are commented... comment for the scenario